### PR TITLE
ExacqTechnologies.exacqVisionClient version 22.12.2.0

### PR DESCRIPTION
**Pull request by camatthew**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98396)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @camatthew,

The installer isn't unattended:
image

Is there an installer switch to bypass this and have the installer perform the install automatically?

File: manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: ExacqTechnologies.exacqVisionClient
+PackageVersion: 22.12.2.0
+Installers:
+- InstallerUrl: https://cdnpublic.exacq.com/22.12/exacqVisionClient_22.12.2.0_x64.msi
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerSha256: A9FFCA61C34E77DF9436BB7036BB71CB8DE427D8543E74F5D7422F665725126C
+  ProductCode: '{133D0A53-AABC-48AE-98DA-B85512C51E78}'
```
Comment (stephengillie) [inline manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml:12]:
```suggestion
  ProductCode: '{133D0A53-AABC-48AE-98DA-B85512C51E78}'
  InstallerSwitches:
    Custom: /quiet
```

Comment (stephengillie) [reply manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml:12]: Using `/S` gives the installer switch menu:
image

**Comment by camatthew** [comment]
Let's see if you comment worked. Thanks for the suggestion, I have been extremely busy with my college class I'm taking write now so I haven't had time to fix this.

File: manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  InstallerSwitches:
+    Custom: /quiet
```
Comment (Trenly) [inline manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml:14]: @stephengillie's suggestion will make it so all installs are silent, even if the user requests an interactive one.
```suggestion
  InstallerSwitches:
    Silent: /quiet
    SilentWithProgress: /passive
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Installers:
+- InstallerUrl: https://cdnpublic.exacq.com/22.12/exacqVisionClient_22.12.2.0_x64.msi
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerSha256: A9FFCA61C34E77DF9436BB7036BB71CB8DE427D8543E74F5D7422F665725126C
+  ProductCode: '{133D0A53-AABC-48AE-98DA-B85512C51E78}'
+  InstallerSwitches:
+    Silent: /quiet
+    SilentWithProgress: /passive
```
Comment (stephengillie) [inline manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml:15]:
```suggestion
```

Comment (stephengillie) [inline manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml:15]:
```suggestion
    Custom: /quiet
```

Comment (stephengillie) [reply manifests/e/ExacqTechnologies/exacqVisionClient/22.12.2.0/ExacqTechnologies.exacqVisionClient.installer.yaml:15]: Testing in a Windows 11 VM, this has been the only combination that results in an unattended install. It appears there may not be a way to display the progress bar with an unattended install.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**